### PR TITLE
[Actions-Salesforce] Update operation choice labels

### PR DESCRIPTION
**Pull request by nick-Ag**
This is a purely cosmetic change which updates the labels for the Salesforce `operation` choices.

## Testing

Screen Shot 2022-06-14 at 1 01 06 PM

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [x] [Segmenters] Tested in the staging environment

### CODE REVIEW

**Review by maryamsharif — APPROVED** [review]

File: packages/destination-actions/src/destinations/salesforce/sf-properties.ts [hunk 1]
```diff
@@ -8,9 +8,9 @@ export const operation: InputField = {
   type: 'string',
   required: true,
   choices: [
-    { label: 'Create', value: 'create' },
-    { label: 'Update', value: 'update' },
-    { label: 'Upsert', value: 'upsert' },
+    { label: 'Create new record', value: 'create' },
+    { label: 'Update existing record', value: 'update' },
+    { label: `Update or create a record if one doesn't exist`, value: 'upsert' },
```
Comment (brennan) [inline packages/destination-actions/src/destinations/salesforce/sf-properties.ts:13]: Maybe "Update an existing record, or create a record if one doesn't exist"? Don't know if that's too wordy.

Comment (kdaswani) [reply packages/destination-actions/src/destinations/salesforce/sf-properties.ts:13]: We had this same discussion @brennan: https://segment.slack.com/archives/C02HKTCVD33/p1655164089390159. Torn on the correct wording! Want to avoid it being too lengthy but don't want to sacrifice clarity. I'm fine with `Update or create a record if one doesn't exist` since it seems to make enough sense IMO.
